plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/natures/DataflowJavaProjectNature.java

```diff
@@ -59,67 +57,32 @@ public void setProject(IProject project) {
    */
   public static void addDataflowJavaNatureToProject(IProject project, IProgressMonitor monitor)
       throws CoreException {
-    if (isProjectAccessible(project)) {
+    Preconditions.checkNotNull(project);
+    if (!project.isAccessible()) {
```

Comment:
Forgot to mention this. The wrongly named old method should have been "isProjectNotAssible()" or its test should have been reversed.